tests/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Test/assets/TestClasses/UA0013.Fixed.cs

```diff
@@ -35,6 +36,25 @@ public class NotAWebController : Foo.ApiController
     {
 
     }
+
+    public partial class Controller2
+    {
+        Controller Controller;
+    }
+
+    public partial class Controller2 : Controller
```

Comment:
When this compiles, is this System.Web.Mvc.Controller or Microsoft.AspNetCore.Mvc.Controller?

Reply:
Likely it won't compile, unfortunately, until the `using System.Web.Mvc` is removed. We could have the code fix do that, but it's possible that other code in the class is using that namespace so it's hard to make this change without breaking something. We have other code fixes to remove those namespaces, so I think this is ok.

Now that you point this out, though, I'm kind of surprised that Roslyn's simplifier removed the namespace qualification here. My hunch is that it only did that because the test doesn't have a System.Web reference and in the real world, it would have been left as `Microsoft.AspNetCore.Mvc.Controller` (unless, of course, UA had previously removed the System.Web.Mvc namespace).

Reply:
I thought the `UsingSystemWebAnalyzer` UA0001 would remove the System.Web.Mvc namespace. We're testing UA0013 but don't all of the analyzers run in the way this is configured?

Reply:
UA0001 will remove the using statement, but in our unit tests we only apply the code fix being tested.